pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+> Open source java application designed for load testing functional behavior and measure performance.
+> More information: <https://jmeter.apache.org>.
+
+- Run jmeter in cli mode for a test plan:
+
+`jmeter -n -t {{name}}.jmx`
```

Comment:
```suggestion
`jmeter --nongui --testfile {{path/to/file}}.jmx`
```

```suggestion
`jmeter --nongui --testfile {{path/to/file.jmx}}`
```

Is `.jmx` a required extension? Or it can be any? Choose the second suggestion if extension is required but it doesn't matter what it is.

Reply:
It is a required extension for jmeter to boot up in this case. 
Please validate! ✅